datacats/scripts.py

```diff
@@ -26,3 +26,4 @@
 DATAPUSHER = SCRIPTS_DIR + '/datapusher.sh'
 INSTALL_POSTGIS = SCRIPTS_DIR + '/install_postgis.sh'
 ADJUST_DEVINI = SCRIPTS_DIR + '/adjust_devini.py'
+UPDATE_ADD_ADMIN = SCRIPTS_DIR + '/update_add_admin.sh'
```

Comment:
let's use
```
os.path.join(SCRIPTS_DIR,  'update_add_admin.sh')
```
here.

It is crazy how many stupid errors due to missing or extra "/" in the file path come from things like this.